Automattic/jetpack
# OpenTable block: proxy requests through new REST API endpoint.

Fixes #22006

#### Changes proposed in this Pull Request:

* OpenTable made changes on their end, and direct requests to get list of restaurants now fail. Going through a proxy goes around the problem for now.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Posts > Add New
* Add a new OpenTable block
* Search for a string
* You should get some restaurant results matching your search.
